hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -161,61 +168,62 @@ This is intended to be unique but also provide structured information to the DHC
+
 Default: ""  
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
 #### MAC_ADDRESS
 Overrides the Ethernet MAC address with the given value. e.g. dc:a6:32:01:36:c2  
+
 Default: ""   
-Version: pieeprom-2020-04-16.bin
+Version: 2020-04-16  
 
 ### DISABLE_HDMI
 The [HDMI boot diagnostics](./boot_diagnostics.md) display is disabled if DISABLE_HDMI=1. Other non-zero values are reserved for future use.
 
 Default: 0   
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
-`Version: pieeprom-2020-06-15.bin - STABLE`
-The `disable_splash` property is no longer checked because the HDMI diagnostics screen is started before config.txt is read.
+From version 2020-09-03 the `disable_splash` property in `config.txt` is no longer checked because the HDMI diagnostics screen is started before `config.txt` is read.
 
 ### ENABLE_SELF_UPDATE
-Allows the bootloader to update itself instead of requiring recovery.bin. This is intended to make it easier to update the bootloader firmware via network boot. To enable set `ENABLE_SELF_UPDATE=1` and add `bootloader_update=1` in config.txt.
-N.B. There is no automatic rollback in the event of a power failure during the firmware update. In the unlikely event of this happening you will have to use Pi Imager to apply the rescue image.
-
-If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the boot partition (or TFTP root). If the update files are different to the current image then the update is applied and system is reset. Otherwise, if the images are identical then boot continues as normal.
+Enables the bootloader to update itself from a network or USB-MSD boot.
 
-Since the updates files are in the same format as generated by rpi-eeprom-update you can use rpi-eeprom-update to install the files to /boot so long as /boot mounts the approprate boot device / tftp-root (via NFS).
+If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the USB MSD boot partition (or TFTP root). If the update image differs from the current image then the update is applied and system is reset. Otherwise, if the EEPROM images are byte-for-byte identical then boot continues as normal.
 
-**The bootloader only reads the configuration from the bootconf.txt in the EEPROM and not the one in the update files. Therefore, in order to enable `ENABLE_SELF_UPDATE` you have to first update the bootloader via the SD-CARD or FLASHROM.**
+Since the updates files are in the same format as generated by rpi-eeprom-update you can use rpi-eeprom-update to install the files to `/boot` so long as `/boot` mounts the approprate boot device / tftp-root (via NFS).
 
-Default: 0  
-Version: pieeprom-2020-04-16.bin  
+Notes:-
+*  Self-update is not enabled in SD boot because the ROM can already load recovery.bin from the SD card. 
```

Comment:
Bit of a nit this one, but I would change ` because` to `; instead`. (With the semicolon). This makes it neutral as to reasoning, since the reason given is not why it hasn't been enabled for SD boot.

Reply:
ok